site/styles/_typography.scss

```diff
@@ -15,55 +15,105 @@
 }
 
 @mixin heading-1-style() {
-  @include typography-style(32px, 0.4px, $font-weight-normal, 8px, var(--body-color));
+  @include typography-style(
+    $font-size: 32px,
+    $letter-spacing: 0.4px,
+    $font-weight: $font-weight-normal,
+    $margin-bottom: 8px,
+    $font-color: var(--body-color),
+    $line-height: 42px
+  );
   @include media-breakpoint-up(xs) {
-    @include typography-style(45px);
+    @include typography-style($font-size: 46px, $line-height: 56px);
```

Comment:
This is pretty nitpicky... but can you do one arg per line wherever the mixin is used with more than one arg?  It will make future diffs much easier to read!

Other than that I'm happy to approve!

Reply:
Do you know which stylelint rule this is? I can't seem to find it. Or should I just format and add ignore?

Reply:
Hmm is Styleline forcing them onto one line?

Reply:
I think this might be what you are looking for: https://stylelint.io/user-guide/rules/list/function-parentheses-newline-inside/

Reply:
No good. I did some investigation and it appears its this prettier config
https://stackoverflow.com/questions/45565587/prettier-js-method-arguments-on-newline-formatting
Putting `// prettier-ignore` before each mixin works though

Reply:
Hmm... I'll just approve since it was only a nitpick anyway.